addons/account/static/src/components/account_payment_field/account_payment.xml

```diff
@@ -69,33 +69,41 @@
                 <div>
                     <table>
                         <tr>
-                            <td><strong>Amount: </strong></td>
-                            <td>
+                            <td class="fw-bolder">Amount: </td>
+                            <td class="ps-1">
                                 <t t-out="props.amount_company_currency"></t>
                                 <t t-if="props.amount_foreign_currency">
                                     (<span class="fa fa-money"/> <t t-out="props.amount_foreign_currency"/>)
                                 </t>
                             </td>
                         </tr>
                         <tr>
-                            <td><strong>Memo: </strong></td>
-                            <td>
-                                <div style="width: 200px; word-wrap: break-word">
+                            <td class="fw-bolder align-top">Memo: </td>
+                            <td class="ps-1 position-relative">
+                                <div class="o_memo_content" t-on-mouseenter="onMemoHover" t-on-mouseleave="onMemoLeave">
+                                    <t t-out="props.ref"/>
+                                </div>
+                                <div class="o_memo_tooltip">
                                     <t t-out="props.ref"/>
                                 </div>
                             </td>
                         </tr>
                         <tr>
-                            <td><strong>Date: </strong></td>
-                            <td><t t-out="props.date"/></td>
+                            <td class="fw-bolder">Date: </td>
+                            <td class="ps-1"><t t-out="props.date"/></td>
                         </tr>
                         <tr>
-                            <td><strong>Journal: </strong></td>
-                            <td><t t-out="props.journal_name"/><span t-if="props.payment_method_name"> (<t t-out="props.payment_method_name"/>)</span></td>
+                            <td class="fw-bolder">Journal: </td>
+                            <td class="ps-1">
+                                <t t-out="props.journal_name"/>
+                                <span t-if="props.payment_method_name">
+                                    (<t t-out="props.payment_method_name"/>)
+                                </span>
+                            </td>
                         </tr>
                         <tr t-if="props.company_name">
-                            <td><strong>Branch: </strong></td>
-                            <td><t t-out="props.company_name"/></td>
+                            <td class="fw-bolder">Branch: </td>
```

Comment:
Another small nitpicking, does the space is needed after the ":" ? (Same for all td that are bolder)

Reply:
Indeed, they're not needed. will update the PR